google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/BreakpointComparer.java

```diff
@@ -75,14 +77,16 @@ public int compare(Breakpoint o1, Breakpoint o2) {
       }
       return s1.getPath().compareTo(s2.getPath());
     }
-    Date d1 = BreakpointUtil.parseDateTime(o1.getFinalTime());
-    Date d2 = BreakpointUtil.parseDateTime(o2.getFinalTime());
-    if (d1 == null) {
+
+    Date d1, d2;
+    try {
+      d1 = ISODateTimeFormat.dateTime().parseDateTime(o1.getFinalTime()).toDate();
```

Comment:
there is a slight behavior change here in the sense that if parsing the first fails we don't parse the second. Not sure if that's important though.